lib/servers/serialserver/serial_server.py

```diff
@@ -151,10 +151,10 @@ def baudrate(self, c, data=None):
         """Sets the baudrate."""
         ser = self.getPort(c)
         if data is None:
-            return long(ser.getBaudrate())
-        else: 
```

Comment:
pyserial changed the API so that baudrate and all the other class attributes (http://pythonhosted.org/pyserial/pyserial_api.html#classes) are accessed with getters and setters.  So serial.getBaudrate() becomes serial.baudrate.  If you want to set the value you do for example serial.baudrate = 9600.